schema/monitoring_metric.json

```diff
@@ -0,0 +1,41 @@
+{
+    "$udmi_version": "1.4.1",
+    "title": "Monitoring metric",
+    "description": "One metric",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "properties": {
+      "timestamp": {
+        "type": "string",
+        "format": "date-time"
+      },
+      "severity": {
+        "enum": [
+          "ERROR",
+          "NONE"
+        ]
+      },
+      "project": {
```

Comment:
Most of the constructs in UDMI use the message envelope (container for the message) to hold the project, location, registry, and device_id -- see envelope.json  -- this is essentially the structure as described here https://cloud.google.com/iot/docs/how-tos/mqtt-bridge  -- again, just something for us to consider to use common constructs when they make sense.